package/gluon-respondd/files/etc/hotplug.d/iface/10-gluon-respondd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec /etc/init.d/gluon-respondd reload
```

Comment:
Ah, I just had a closer look at `base-files/files/sbin/hotplug-call`, they are sourced in a sub-shell.